src/accounts.rs

```diff
@@ -78,6 +78,14 @@ impl Accounts {
         self.accounts.read().await.get(&id).cloned()
     }
 
+    /// Get the currently selected account's id or None if no account is selected
```

Comment:
See https://rust-lang.github.io/rfcs/0505-api-comment-conventions.html

> The summary line should be written in third person singular present indicative form. Basically, this means write "Returns" instead of "Return".

Also the dot in the end is missing.

Overall, this should be "Returns the currently selected account's id or None if no account is selected." if we follow rust API conventions.